org_fedora_oscap/gui/spokes/oscap.py

```diff
@@ -402,7 +399,7 @@ def _fetch_data_and_initialize(self):
             self._fetching = True
 
         thread_name = None
-        if self._policy_data.content_url and self._policy_data.content_type != "scap-security-guide":
+        if self.data_handler.needs_fetch_content():
```

Comment:
I suggest to rename `needs_fetch_content` to `content_is_fetchable` or something like that, because the need arises from the fact that we don't have it, which may not be the case.